compiled_starters/clojure/codecrafters.yml

```diff
@@ -7,5 +7,5 @@ debug: false
 # Use this to change the Clojure version used to run your code
 # on Codecrafters.
 #
-# Available versions: clojure-1.11.3
-language_pack: clojure-1.11.3
+# Available versions: clojure-1.10.3
+language_pack: clojure-1.10.3
```

Comment:
Not sure what's happening here - we're changing the default to an older version, and we're renaming `clojure-1.11.3.Dockerfile` to `clojure-0.Dockerfile`?

Reply:
Ah, ignore - just saw your comment